test/README.md

```diff
@@ -94,7 +94,7 @@ environment that meets
 to specify the build tag `e2e`.
 
 ```bash
-go test -v -tags=e2e -count=1 ./test/e2e
+go test -v -tags=e2e -count=1 ./test/e2e -tag e2e
```

Comment:
What does `-tag e2e` do? Why is it distinct from `-tags-e2e`? Add the reason to the description above.

Reply:
It's probably better to run tests against `latest` so I removed `-tag` and added a description on how to override this behavior. Also `upload-test-images.sh` does not need tag images when using dockerhub